pkg/network/node/ovs/ovs.go

```diff
@@ -35,6 +35,16 @@ type Interface interface {
 	// error if the interface is not currently a bridge port.)
 	DeletePort(port string) error
 
+	// AddGroup adds a group to the bridge
+	AddGroup(groupID uint32, groupType string, buckets []string) error
```

Comment:
`AddGroup` and `DeleteGroup` can and should be done as part of `Transaction`

Reply:
done